src/main/java/Task.java

```diff
@@ -0,0 +1,17 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718");

```

Comment:
Perhaps you may want to consider putting the unicode characters as constants for easier reference?